Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -73,6 +73,8 @@ def bottle_args
       switch "--only-json-tab",
              depends_on:  "--json",
              description: "When passed with `--json`, the tab will be written to the JSON file but not the bottle."
+      flag   "--committer=",
+             description: "Specify a committer in the format 'Name <email@example.com>'."
```

Comment:
This syntax doesn't render correctly when using `brew bottle --help`, and underlines the email portion instead (`ronn` syntax). Shall I replace the angular brackets with some other character like parentheses, or use two flags (`--committer-name` and `committer-email`) instead?

I think the improper rendering in the usage text and manpages could lead to confusion and is undesirable.

Reply:
Referring to the format as git's [standard author format](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---authorltauthorgt), rather than providing an example.